### DISCUSSION TITLE
Les presento mi estructura de la practica de la pastelería - Mazun

### DISCUSSION THREAD

**Discussion by LaMazorca** [Launch-X-Latam/MisionFrontEnd#263] [Show and tell]
Hola compañeros les dejo mi repositorio de la pastelería que he realizado hasta ahora, tengo el plan de hacer el CSS también, pero aún no he resuelto como hacerlo, de igual forma hay algunas ideas que tengo que me gustaría realizar, pero se que necesito saber más que solo HTML y CSS para hacerlo, así que esperare a hasta lo demás del curso, agradezco sus comentarios y sugerencias, me ha encantado todo este programa y me encantaría tener contacto con ustedes después he aprendido incluso hasta de ustedes. Gracias.

[Repositorio de la pastelería](https://lamazorca.github.io/LaunchX-Pasteleria/)

**Comment by brayandoria98** [comment]
Creo que le link al repositorio está mal, pero pude entrar igual y me parece bien lo que has hecho y como tú no le haré nada en CSS hasta que lo sepa bien. Igual es una práctica de HTML aún.

**Reply by LaMazorca** [reply]
Gracias, no me había fijado, veré cambiar el link
